src/callback/event_callbacks.cpp

```diff
@@ -0,0 +1,47 @@
+#include <fmod_studio.hpp>
+#include "event_callbacks.h"
+#include "../godot_fmod.h"
+
+namespace Callbacks{
+    std::mutex callback_mut;
+
+    FMOD_RESULT F_CALLBACK eventCallback(FMOD_STUDIO_EVENT_CALLBACK_TYPE type, FMOD_STUDIO_EVENTINSTANCE *event, void *parameters) {
+        auto *instance = (FMOD::Studio::EventInstance *) event;
+        auto instanceId = (uint64_t) instance;
```

Comment:
Note for the future, we should use `uintptr_t` to handle ptr to int conversions (and avoid C cast in favor of reinterpret).
I wonder if this would help on #47

Reply:
Probably yes, but it's a rework I keep for the next FMOD major version.